lib/Xmds/Soap5.php

```diff
@@ -95,11 +96,14 @@ public function RegisterDisplay($serverKey, $hardwareKey, $displayName, $clientT
                 $displayElement->setAttribute('status', 0);
                 $displayElement->setAttribute('code', 'READY');
                 $displayElement->setAttribute('message', 'Display is active and ready to start.');
-                $displayElement->setAttribute('version_instructions', $display->versionInstructions);
 
                 // Display Settings
-                $settings = $display->getSettings();
-
+                if (isset($this->display->overrideConfig))
```

Comment:
refactor - should be just $this->display->getSettings();`

we always want to consider overrides here?